[ENH] Document how to give feedback on documentation and add issue template (#132)
neurobagel/documentation: docs/getting_help.md

# Getting Help

## Report a bug or request a feature
For a bug or feature request, feel free to open an issue in the repository for the relevant tool in the [Neurobagel GitHub organization](https://github.com/neurobagel).

If you are using one of our web tools ([https://annotate.neurobagel.org](https://annotate.neurobagel.org) or [https://query.neurobagel.org](https://query.neurobagel.org), you can also submit suggestions and bugs using the feedback widget directly on the site.

## Documentation feedback

> Neurobagel is always looking to improve their user documentation.

Reviewer: ```suggestion
We are always looking to improve our user documentation.
```
This is us talking about ourselves, no?

> To request a change or addition to existing documentation, please [open an issue in the `neurobagel/documentation` repo](https://github.com/neurobagel/documentation/issues).

Reviewer: I'm guessing there is no easy way to link directly to the nice new issue template you made?

Author: I could link to this page https://github.com/neurobagel/documentation/issues/new/choose which should update to show the new issue template once the PR is merged.

However, I think I would still prefer to direct the user to the list of current issues here so they can first see which topics/changes have already been requested, etc.

Let me know if you feel differently.

Reviewer: good point, agreed
